apps/andi/lib/andi_web/live/organization_live_view.ex

```diff
@@ -0,0 +1,126 @@
+defmodule AndiWeb.OrganizationLiveView do
```

Comment:
I'd love to see us take a crack at making this not repeat some of the problems with the datasets page (Ex. use the Repo + Query to do searching as it's arguably faster or at least doesn't kill the client machine with javascript based filtering). However, I'm fine with pushing forward if that's too much to pack on there.

Reply:
yeah good call